examples/peoples/index.html

```diff
@@ -49,6 +60,11 @@
 	</head>
 	<body>
 		<div id="target" />
+        <div onclick="hideMe(this)" id="description">
```

Comment:
Please follow the rest of the code and use tabs, not spaces for indentation.

Reply:
"It's not me, it's my editor"

Reply:
(how many spaces do you map tabs to? 2? 3? 4?)

Reply:
hmm. weird. i'm doing this live on my server so it wasn't through my normal editor.

Reply:
Why does it matter how many spaces I map tabs to? Use tab characters, not spaces, for indentation.